controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go

```diff
@@ -275,6 +287,11 @@ func (r *ClusterReconciler) removeAkoPackageRefFromClusterBootstrap(ctx context.
 		}
 	}
 
+	// Remove skip deleting ako packageinstall annotation to clusterboostrap if it exists
```

Comment:
This is not right, we should remove the annotation when avi resource clean up finished